docs/_pages/papers.md

```diff
@@ -58,7 +58,7 @@ We add paper to this list if we want to presente it later, or want to ask other
 
 | Seminar | Member | Paper |
 | ------- | ------ | ----- |
-| [S2E15](https://ai-ml.club/events/seminar-meeting-minutes-2-15/) | [@VDeamoV](https://github.com/VDeamoV) | [Progressive Pose Attention Transfer for Person Image Generation](https://arxiv.org/pdf/1904.03349) |
+| [S2E15](https://ai-ml.club/events/seminar-meeting-minutes-2-15/) | [@VDeamoV](https://github.com/VDeamoV) | [ Progressive Pose Attention Transfer for Person Image Generation](https://arxiv.org/pdf/1904.03349) |
```

Comment:
Space in links is forbidden by the linting system.

```
docs/_pages/papers.md: 61: MD039/no-space-in-links Spaces inside link text [Context: "[ Progressive Pose Attention T..."]
```

Please remove the additional space, then I believe this PR will pass the CI test as well. :)